### ISSUE TITLE
Feature request: expand :rocket: and other GitHub text emoticons

### ISSUE THREAD

**Issue by dimaqq** [hynek/hatch-fancy-pypi-readme#49]
Quite a few folks (myself, my coworkers, and even pydantic maintainers) tend to use `:rocket:` and similar text emoticons from GitHub-flavoured markdown in the readme files.

When these files are packed into a fancy readme and pushed to PyPI, the emoticons are not displayed properly.

I think it would be awesome if this tool took care of the inconsistency.

Example:

![Screenshot 2024-06-20 at 13 38 30](https://github.com/hynek/hatch-fancy-pypi-readme/assets/662249/1a087d37-3556-421f-aeb0-1f68909fd2c0)
![Screenshot 2024-06-20 at 13 38 47](https://github.com/hynek/hatch-fancy-pypi-readme/assets/662249/9980472e-0d0d-4c9f-b974-8a35fb0323d3)

**Comment by hynek** [comment]
Yeah I noticed that in Pydantic's PyPI release but since nobody approached me, I didn't tackle it yet since I wasn't sure anybody cares about that at all.

I _think_ there's an official list of GitHub text emoticons somewhere?

**Comment by dimaqq** [comment]
Not sure, but there are unofficial lists like this one https://gist.github.com/rxaviers/7360908

No clue as to veracity of this specific list

**Comment by dimaqq** [comment]
Maybe this is better: 

https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md

**Comment by hynek** [comment]
ha: https://api.github.com/emojis

that's lots of emojis tho! there **has** to be some tool that translates them automatically, though?
